x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -353,80 +356,440 @@ export default ({ getService }: FtrProviderContext): void => {
+            supertest,
+            params: {
+              cases: [
+                {
+                  id: postedCase.id,
+                  version: postedCase.version,
+                  status: CaseStatuses['in-progress'],
+                },
+              ],
+            },
+            auth: caseAuth,
+          });
+
+          const signals = await createSecuritySolutionAlerts(supertest, log);
+
+          const alert = signals.hits.hits[0];
+          expect(alert._source?.[ALERT_WORKFLOW_STATUS]).eql('open');
+
+          await createCommentAndRefreshIndex({
+            caseId: postedCase.id,
+            alertId: alert._id,
+            alertIndex: alert._index,
+            expectedHttpCode: attachmentExpectedHttpCode,
+            auth: attachmentAuth,
+          });
+
+          const updatedAlert = await getSecuritySolutionAlerts(supertest, [alert._id]);
+
+          expect(updatedAlert.hits.hits[0]._source?.[ALERT_WORKFLOW_STATUS]).eql(
+            expectedAlertStatus
+          );
+        };
+
+        const bulkCreateAlertsAndVerifyCaseIdsInAlertSchema = async (totalCases: number) => {
+          const cases = await Promise.all(
+            [...Array(totalCases).keys()].map((index) =>
+              createCase(supertest, {
+                ...postCaseReq,
+                settings: { syncAlerts: false },
+              })
+            )
+          );
+
+          const signals = await createSecuritySolutionAlerts(supertest, log);
+          const alert = signals.hits.hits[0];
+
+          for (const theCase of cases) {
+            await createCommentAndRefreshIndex({
+              caseId: theCase.id,
+              alertId: alert._id,
+              alertIndex: alert._index,
+            });
+          }
+
+          const updatedAlert = await getSecuritySolutionAlerts(supertest, [alert._id]);
+          const caseIds = cases.map((theCase) => theCase.id);
+
+          expect(updatedAlert.hits.hits[0]._source?.[ALERT_CASE_IDS]).eql(caseIds);
+
+          return { updatedAlert, cases };
+        };
+
+        it('should change the status of the alert if sync alert is on', async () => {
+          await bulkCreateAlertsAndVerifyAlertStatus({
+            syncAlerts: true,
+            expectedAlertStatus: 'acknowledged',
+          });
+        });
+
+        it('should NOT change the status of the alert if sync alert is off', async () => {
+          await bulkCreateAlertsAndVerifyAlertStatus({
+            syncAlerts: false,
+            expectedAlertStatus: 'open',
+          });
+        });
+
+        it('should change the status of the alert when the user has read access only', async () => {
+          await bulkCreateAlertsAndVerifyAlertStatus({
+            syncAlerts: true,
+            expectedAlertStatus: 'acknowledged',
+            caseAuth: {
+              user: superUser,
+              space: 'space1',
+            },
+            attachmentAuth: { user: secOnlyReadAlerts, space: 'space1' },
```

Comment:
I think the reason that this user fails to when they don't have index privileges is because we're trying to update the status of the alert in this test:

```
Failed to update alert status ids: [{"id":"155bf8908ab2dee36b72f6e304fcd7de3de53104b3f8fb351f4e0463a704e63d","index":".internal.alerts-security.alerts-default-000001","status":"in-progress"}]: ResponseError: security_exception
```